### ISSUE TITLE
In advanced search screen, when selecting more than 1 subvocabulary, the query fails

### ISSUE THREAD

**Issue by tfrancart** [NatLibFi/Skosmos#658]
## At which URL did you encounter the problem?

http://finto.fi/yso/en/search?clang=en&q=agriculture&vocabs=

## What steps will reproduce the problem?
1. Go to above URL. Note that the search yields 2 results from YSO vocabulary
2. In the left dropdown "By subvocabulary", select vocabulary "YSO - General Finnish Ontology", and click on "limit search"
3. The list is refreshed
4. Now select a second subvocabulary from the list (http://finto.fi/yso/en/search?clang=en&q=agriculture&vocabs=), and click on "limit search"
5. The query fails and return no results

## What is the expected output? What do you see instead?

Expected output : the same 2 concepts in the search results.
What happens instead : no results found.

The SPARQL query is wrong, the URI of the 2 graphs are not properly concatenated (exemple taken from another Skosmos install with different graph URIs). Note the "+" signs between the 2 URIs :

```
...
  {
   ?s a ?type .
    ?s skos:inScheme <http://data.education.fr/voc/scolomfr/scolomfr-voc-022+http://data.education.fr/voc/scolomfr/scolomfr-voc-023> .
  }`
```

**Comment by osma** [comment]
Thanks for reporting!

**Comment by osma** [comment]
This was fixed by PR #664
